src/VersionManager.cs

```diff
@@ -158,40 +253,45 @@ private static string GetInvoiceNumber()
         /// </summary>
         /// <param name="version">The version number required.</param>
         /// <returns>The path to the documentation, or null, if it cannot be provided.</returns>
-        internal static string GetDocumentationPath(string version)
+        public static string GetDocumentationPath(Version version)
         {
-            string fullPath = Path.Combine(USER_DR_DIR, "documentation", version);
+            return Path.Combine(USER_DR_DIR, "documentation", version.ToString());
```

Comment:
Because of the change in behavour here running darkrift docs with versions not availab e leads to some very strange results.

```
darkrift docs 0.0.0 --local
```